tests/nested/manual/uc-grub-boot-chains/modify-gadget.py

```diff
@@ -0,0 +1,48 @@
+import argparse
+import yaml
+
+parser = argparse.ArgumentParser()
+parser.add_argument('version_bump', type=int)
```

Comment:
edition_bump perhaps?

also I think this argument would be clearer as an option instead of a positional arg

Reply:
Done.